examples/suit_update/README.md

```diff
@@ -33,7 +33,7 @@ Table of contents:
 
 - Install python dependencies (only Python3.6 and later is supported):
 
-      $ pip3 install --user cbor2 cryptography
+      $ pip3 install --user cbor2 cryptography LinkHeader
```

Comment:
Any reason not to just install `aiocoap[linkheader] >= 0.4.1`?

Reply:
I didn't know this syntax, what does it do?

Reply:
Ok, changed it.